mgmt/api/INKMgmtAPI.cc

```diff
@@ -1865,8 +1865,7 @@ TSGetErrorMessage(TSMgmtError err_id)
 tsapi TSMgmtError
 TSEncryptPassword(char *passwd, char **e_passwd)
```

Comment:
I agree with that. Do we need API review to remove? This looks like one of TS API.